src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -166,6 +166,21 @@ def get_tables_to_migrate(self, tables_crawler: TablesCrawler) -> Collection[Tab
 
         return Threads.strict("checking all database properties", tasks)
 
+    def get_tables_to_set_acls(self, tables_crawler: TablesCrawler) -> Collection[TableToMigrate]:
```

Comment:
Avoid types in names:

```suggestion
    def get_tables_to_acls(self, tables_crawler: TablesCrawler) -> Collection[TableToMigrate]:
```

Reply:
There is no mention of `acls` in this method, why is it in the name?

Reply:
why not reuse a method like `get_tables_to_migrate`?

Reply:
Because the scope is different.

Reply:
Done